app/main/views/direct_award.py

```diff
@@ -38,7 +43,7 @@ def list_projects():
     )
 
     return jsonify(
-        projects=[project.serialize() for project in projects.items],
+        projects=[project.serialize(with_users='users' in includes) for project in projects.items],
```

Comment:
We could maybe factor `'users' in includes` out to a `with_users` variable at the first point you use it, just to make this read more easily.

Reply:
I actually went the other way, and changed from your suggest to what it is above. But thinking about it again, for readability, I think you're right. Will update.